runtime/runtime/src/lib.rs

```diff
@@ -2142,14 +2142,9 @@ impl Runtime {
 impl ApplyState {
     fn own_congestion_info(
         &self,
-        protocol_version: ProtocolVersion,
+        _protocol_version: ProtocolVersion,
```

Comment:
Remove unused parameter `_protocol_version`. Change return type of function to `Result<CongestionInfo, RuntimeError>`.

Update the call sites and follow through functions to also take in parameter `CongestionInfo` instead of `Option<CongestionInfo>`